docker/backend_httpd/Dockerfile

```
@@ -1,10 +1,19 @@
-# Crafted with the help of the SCLOrg docs:
-# https://github.com/sclorg/nginx-container/tree/master/1.20#3-prepare-an-application-inside-a-container
-
-FROM quay.io/centos7/nginx-120-centos7:centos7
+FROM registry.fedoraproject.org/fedora:43
 
 MAINTAINER copr-devel@lists.fedorahosted.org
```

Hadolint (2.15.1)

[error] MAINTAINER is deprecated

(DL4000)